sssom/cli.py

```diff
@@ -15,11 +15,33 @@
 import logging
 from pandasql import sqldf
 
```

Comment:
@hrshdhgd here, you can do something like this:

```suggestion
input_option = click.option('-i', '--input', help=help_input)
```

Later, inside your commands, you can use `@input_option` instead of `@click.option('-i', '--input', help=help_input)`